src/action/grpc.js

```diff
@@ -64,6 +64,26 @@ class GrpcAction {
     this._store.lndReady = true;
   }
 
+  /**
+   * This is called to initialize the GRPC client to autopilot. Once `autopilotReady`
+   * is set to true on the store GRPC calls can be made to the client.
+   * @return {Promise<undefined>}
+   */
+  async initAutopilot() {
+    await this._sendIpc('lndAtplInit', 'lndAtplReady');
+    log.info('GRPC autopilotReady');
+    if (this._store.settings.autopilot) {
+      try {
+        await this.sendAutopilotCommand('modifyStatus', {
+          enable: true,
+        });
+      } catch (err) {
+        log.error('Failed to activate autopilot', err);
+      }
+    }
+    this._store.autopilotReady = true;
```

Comment:
I'm thinking we should move `grpc.initAutopilot()` and `setting.toggleAutopilot()` into its own `autopilot.js` action module. Feels like we're creating duplicate try/catch code for both here and we should create a common wrapper around the grpc call.

Reply:
fixed